webapp-django/crashstats/crashstats/views.py

```diff
@@ -1050,14 +1049,23 @@ def crashes_per_user(request, default_context=None):
     default = request.session.get('date_range_type', 'report')
     context['date_range_type'] = params.get('date_range_type') or default
 
-    api = SuperSearchUnredacted()
-    # XXX HACK! Because of a bug in SuperSearch we can't search for
-    # '4X.0b' so we have to change that to '4X.0b1' :(
-    # See https://bugzilla.mozilla.org/show_bug.cgi?id=1197251
+    params['original_versions'] = params['versions'][:]  # make a copy
```

Comment:
This takes a copy of what the user expected. 
During the course of this function we're transform `params['versions']` and I want an original copy to be displayed in the template rendering (and CSV outputter).